tests/test_lua.lua

```diff
@@ -0,0 +1,23 @@
+--
+-- tests/test_lua.lua
+-- Automated test suite for Lua base functions.
+-- Copyright (c) 2008 Jason Perkins and the Premake project
+--
+
+	local suite = test.declare("lua")
+
+--
+-- loadfile with custom env
+--
+
+	function suite.loadfile_with_env()
```

Comment:
As it seems there is a special handling for with and without env, an additional test without env would be fine